project/Build.scala

```diff
@@ -20,47 +20,44 @@ object Build extends BuildDef with BuildCommon {
   import BuildSettings._
 
   private val schemaVersions = Seq("3.3", "3.4", "3.5", "3.6")
-
   lazy val mainProjects: Seq[Project] = schemaVersions.map(nitfProject)
-  lazy val testProjects: Seq[Project] = schemaVersions.map(testProject)
-  lazy val leafProjects: Seq[Project] = mainProjects ++ testProjects
 
-  override def projects: Seq[Project] = rootProject.toSeq ++ leafProjects
+  override def projects: Seq[Project] = rootProject.toSeq ++ mainProjects
 
   override def rootProject = Some(
-    Project(id = Metadata.projectName, base = file("."))
-      .aggregate(leafProjects.map(Project.projectToRef): _*)
-      .settings(commonSettings ++ releaseSettings ++ disabledPublishingSettings)
+    Project(id = "root", base = file("."))
+      .aggregate(mainProjects.map(Project.projectToRef): _*)
+      .settings(commonSettings ++ disabledPublishingSettings)
+      .settings(
+        Compile / sources := Seq.empty,
+        Test    / sources := Seq.empty,
+        releaseVcsSign := true,
+        releaseCrossBuild := true,
+        releaseCommitMessage += "\n\n[ci skip]",
```

Comment:
does this have any specific meaning?

Reply:
It means that Travis CI shouldn't run a CI job for the release and post-release commits (which only include a change to `version.sbt` to set and increment the version).